### PR TITLE
ui: overlap in subject pills & long email fix

### PR DESCRIPTION
**Pull request by mitsosf**

### CODE REVIEW

**Comment by mitsosf** [comment]
References #862 & #834

**Comment by mitsosf** [comment]
The subjects are now properly aligned (1) and long emails don't break the style anymore (2):
image

**Comment by ntarocco** [comment]
I tested it out and it looks much better thank you!
There is a point where it breaks, maybe worth it to slightly change the media query?
See:
Screenshot 2021-06-18 at 17 32 16

File: invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less [hunk 1]
```diff
@@ -46,6 +46,17 @@ nav {
   padding-bottom: 8px;
 }
 
+@media only screen and (min-width: 745px) and (max-width: 1000px){
```
Comment (ntarocco) [inline invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/navbar.less:49]: I tested with `1200px` and it looks like is fixes the issues:
```suggestion
@media only screen and (min-width: 745px) and (max-width: 1200px){
```
WDYT?

**Comment by mitsosf** [comment]
@ntarocco 
I also thought that someone may have a very long email, even longer than the one you used for the review. In that case, it may make sense to cut the last part of the email address or something eg.

superlongemailthatwouldtakealotofspace@longemail.com

Could be 
superlongemailthatwouldtakealoto...
or
superlongemail...@longe....com

Not sure if it's worth the efforts/resources, was just thinking about limits

**Comment by ntarocco** [comment]
> @ntarocco
> I also thought that someone may have a very long email, even longer than the one you used for the review. In that case, it may make sense to cut the last part of the email address or something eg.
> 
> [superlongemailthatwouldtakealotofspace@longemail.com](mailto:superlongemailthatwouldtakealotofspace@longemail.com)
> 
> Could be
> superlongemailthatwouldtakealoto...
> or
> superlongemail...@longe....com
> 
> Not sure if it's worth the efforts/resources, was just thinking about limits

You are right indeed. I would keep for the moment like this, and if we have issues reported we will probably do what you suggest! :)

**Pull request merged**